pkg/cmd/roachtest/tests/libpq_blocklist.go

```diff
@@ -38,7 +71,6 @@ var libPQBlocklist20_2 = blocklist{
 	"pq.TestCopyInTypes":                             "5807",
 	"pq.TestCopyRespLoopConnectionError":             "5807",
 	"pq.TestEncodeAndParseTs":                        "41563",
-	"pq.TestErrorDuringStartup":                      "41551",
```

Comment:
is this intentional (does it close issues on 21_1?) that said, i guess at some stage we should cleanup all these older version blocklists because they're per branch now (?)